### DISCUSSION TITLE
Will State Machine Workflows  and  BPMN Workflows release on 3.0

### DISCUSSION THREAD

**Discussion by Tim1104** [elsa-workflows/elsa-core#4434] [Q&A]

**Comment by sfmskywalker** [comment]
Hi @Tim1104 

State machine and BPMN 2.0 will not be included with the initial 3.0 release in order to not delay its release any further. And although there’s no current time line either for either of these features, I hope to be able to start with them sometime after the release. Probably one. Of the first next things we’ll be working on is Multitenancy and BPMN 2.0.
